tenable/io/v3/vm/api.py

```diff
@@ -37,6 +39,14 @@ class VulnerabilityManagement(APIEndpoint):  # noqa: PLR0904
     i.e assets, agents, scanners etc.
     '''
 
+    @property
```

Comment:
@milanaghera-tenable Can you please keep this consistent with the import order?

Reply:
done.